packages/frontend/src/pages/scaling/projects/props/getProjectDetails.ts

```diff
@@ -88,29 +86,43 @@ export function getProjectDetails(
       })
     }
 
-    technologySections.forEach((section) =>
-      items.push({
-        type: 'TechnologySection',
-        props: {
-          items: section.items,
-          id: section.id,
-          title: section.title,
-          isUnderReview: section.isUnderReview,
-        },
-      }),
-    )
+    /* We want state derivation to be after technology section
```

Comment:
How can I test this? Which preview should I look at?

Reply:
There is no state derivation section currently in config, you would have to add it manually